### DISCUSSION TITLE
Problems with using SLEAP models in Bonsai. Single instance model

### DISCUSSION THREAD

**Discussion by KostasTsar** [talmolab/sleap#1294] [Help!] [answered]
Hello everyone,

I've been facing issues trying to run SLEAP models in Bonsai. I am posting here since I expect I am doing something wrong on my end, apologies if it should be on a different forum.

I have in this example a single instance model trained in SLEAP v1.2.9, which has no problem predicting the pose of my animal when run in SLEAP. After I export the model and run it in Bonsai, I am getting no errors and inference is technically running, but the predictions are at random locations. Below is an example.

Prediction example on  single frame in SLEAP:
<img width="1635" alt="sleap_prediction_example_same_frame" src="https://user-images.githubusercontent.com/84953213/235572634-98030218-222d-41af-85a0-92369c033cbf.png">

Preciction example on single frame in Bonsai:
<img width="1902" alt="bonsai_sleap_prediction_example" src="https://user-images.githubusercontent.com/84953213/235572625-ab7ccec0-c8f7-412f-a38c-17a70246be57.png">

The nodes jitter around at random points, but never manage to find the animal.
I am simply giving to the model the same input video as was used in training, without applying any modifications and I am getting no errors, simply completely incorrect predictions.

<img width="209" alt="bonsai_sleap_simple_pipeline" src="https://user-images.githubusercontent.com/84953213/235574770-d438332b-862c-48ed-8f16-cbcedfdcbf9c.png">

Any ideas on how to fix this?

Thank you in advance,
Kostas

**Comment by bruno-f-cruz** [comment, marked as answer]
I suspect you might have the wrong color format specified. You can try to set the `ColorConversion` property in the `PredictSinglePose` to  `Bgr2rgb`. You can also just add a `ConvertColor` operator and use the same conversion string.

**Reply by KostasTsar** [reply]
That was it... Thank you Bruno!
